### DISCUSSION TITLE
"Uphill" requires a provisioning profile. Select a provisioning profile in the Signing & Capabilities editor.

### DISCUSSION THREAD

**Discussion by sharubin** [codemagic-ci-cd/codemagic-docs#1957] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

<img width="838" alt="Снимок экрана 2023-08-09 в 15 55 55" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/96702199/48cef25c-9cdb-4181-8e3c-0f79eb06341c">

### Expected results

I don't have this team on my personal machine, I have no possibility to add it manually to signing&capability section

### Actual results

error

**Comment by IKNikolov** [comment]
@sharubin I have the same issue. I'm trying to build iOS app from an empty (new) react native project and I having the same issue. Do you find some solution for your case?

P.S. With the android build I have no issue.

**Reply by sharubin** [reply]
По-моему я просто в bundle вписал актуальный и заработало, насколько я помню
